container/kubernetes/onboarding/config.json

```diff
@@ -0,0 +1,54 @@
+{
+	"data_directory": "./data",
+	"listeners": {
+		"rest_port": 8000,
+		"udp_port": 41234,
+		"tcp_port": 7070
+	},
+	"receivers": {
+		"udp_port": 41235,
+		"udp_address": "0.0.0.0"
+	},
+	"logger": {
+		"level": "info",
+		"path": "/tmp/",
+		"max_size": 134217728
+	},
+	"default_connector": "rest+ws",
+	"connector": {
+		"rest": {
+			"host": "platform.industry-fusion.com",
```

Comment:
why is the config there?  It should be copied from the root config directory
Also, there is no place for specific URLs in the configs, except for CI/CD test instances

Reply:
Should i get rid of config.json file from this repo , It should be copied from the root config directory ? means which directory (https://github.com/mattmikulina/IndustryFusion-machine-configs/blob/master/OISP%20deployment/platform-launcher/kubernetes/iot-agent/global-config/config.json)